usaspending_api/search/v2/views/spending_over_time.py

```diff
@@ -79,23 +79,29 @@ def validate_request_data(json_data: dict) -> dict:
 
     def database_data_layer_for_subawards(self) -> tuple:
         queryset = subaward_filter(self.filters)
-        obligation_column = "amount"
-        values = ["fy"]
+        obligation_column = "subaward_amount"
 
-        if self.group == "month":
-            queryset = queryset.annotate(month=FiscalMonth("action_date"), fy=FiscalYear("action_date"))
-            values.append("month")
+        # Note: SubawardSearch already has an "fy" field that corresponds to the prime award's fiscal year.
+        #       This, however, needs "fy" to be the fiscal year of the sub_action_date (i.e. "sub_fiscal_year").
+        #       And so, Django gets confused simply aliasing it to "fy" as "fy" is already a field in the model.
+        #       To get around that, we're doing this little dance of annotate() and values().
+        queryset = queryset.annotate(primel_fy=F("fy"))
```

Comment:
should `primel_fy` have an `l` in it, or is that a typo? Should it just be `prime_fy`? Its referenced below, so if you change it, change it there too.

Reply:
Nope, that's a typo. Fixing